packages/support/src/tools/_mixins.button.scss

```diff
@@ -0,0 +1,7 @@
+@mixin base-button {
```

Comment:
I'm not seeing this used anywhere else. Do you have a plan for this?

Reply:
Good catch – this is a leftover from how I was approaching the close button originally. Removed 👍